niceman/retrace/packagemanagers.py

```diff
@@ -179,6 +271,40 @@ def _create_package(self, pkgname):
         lgr.debug("Found package %s", pkg)
         return pkg
 
+    class MultipleReleaseFileMatch(RuntimeError):
+        pass
+
+    def _find_release_date(self, site, archive):
+        if not site:
+            return None
+        # First find the release file
+        # This uses logic similar to apt.utils.get_release_filename_for_pkg
+        # but restricts matches to the source site
+        rfile = None
+        dirname = apt_pkg.config.find_dir("Dir::State::lists")
```

Comment:
I think we need to init apt_pkg first or it then returns incorrect path -- or what am I doing wrong? ;)
```shell
hopa:~
$> python -c 'import apt_pkg; print apt_pkg.config.find_dir("Dir::State::lists")'

/
3 10836.....................................:Mon 20 Feb 2017 01:18:24 PM EST:.
hopa:~
$> python -c 'import apt_pkg; print apt_pkg.config.find_dir("Dir::State")'       
/                                                                      
3 10837.....................................:Mon 20 Feb 2017 01:18:29 PM EST:.
hopa:~
$> python -c 'import apt_pkg; apt_pkg.init(); print apt_pkg.config.find_dir("Dir::State")' 
/var/lib/apt/
3 10838.....................................:Mon 20 Feb 2017 01:19:05 PM EST:.
hopa:~
$> welp apt_pkg       
PATH       : /usr/lib/python2.7/dist-packages/apt_pkg.x86_64-linux-gnu.so
SRC PATH   : /usr/lib/python2.7/dist-packages/apt_pkg.x86_64-linux-gnu.so
VERSION    : Not found
__version__: No __version__
PACKAGE    : python-apt
ii  python-apt     1.4.0~beta2  amd64        Python interface to libapt-pkg
python-apt:
  Installed: 1.4.0~beta2
  Candidate: 1.4.0~beta2
  Version table:
 *** 1.4.0~beta2 900
        900 http://http.debian.net/debian stretch/main amd64 Packages
        600 http://http.debian.net/debian sid/main amd64 Packages
        100 /var/lib/dpkg/status
```